android/src/main/java/io/kommunicate/app/RNKommunicateChatModule.java

```diff
@@ -499,6 +502,41 @@ public void createSettings(final String setting) {
         FileUtils.writeSettingsToFile(currentActivity, setting);
     }
 
+    @ReactMethod
+    public void updateDefaultSetting(final ReadableMap settingMap, final Callback callback) {
+        final Activity currentActivity = getCurrentActivity();
+try {
+                KmSettings.clearDefaultSettings();
+                if (settingMap.hasKey("defaultAgentIds")) {
+                    List<String> agentList = new ArrayList<String>();
+                    for(int i = 0; i < settingMap.getArray("defaultAgentIds").size(); i++){
+                        agentList.add(settingMap.getArray("defaultAgentIds").getString(i));
```

Comment:
Can't we add all of them in one go instead of loop by converting the array to list?